app/models/lot.rb

```diff
@@ -0,0 +1,7 @@
+class Lot < ApplicationRecord
+  validates :title, presence: true, length: { minimum: 2, maximum: 24 }
```

Comment:
Why maximum size of title can be 24?
You don't want lot's titles be like Aliexpress's product's titles? :D